### DISCUSSION TITLE
Can't uploading Image: Redirecting to s3.ap-south-1.amazonaws.com. But my object storage is linode

### DISCUSSION THREAD

**Discussion by imran-stacklearner** [outline/outline#5831] [Self Hosting]
AWS_S3_UPLOAD_BUCKET_NAME: stack-outline
  AWS_S3_UPLOAD_BUCKET_URL: https://xxxxxxxxxx-1.linodeobjects.com
  AWS_S3_ACCELERATE_URL: https://xxxxxxxxxxx.linodeobjects.com
  AWS_S3_UPLOAD_MAX_SIZE: "26214400"
  AWS_S3_FORCE_PATH_STYLE: "false"
![image](https://github.com/outline/outline/assets/141145172/f0ea6b7e-2d86-4a8a-8a73-4e8e1b5af2b3)

**Comment by dwbrite** [comment]
Did you find a solution? I'm seeing something similar.
